lib/Elastica/Util.php

```diff
@@ -197,6 +197,6 @@ public static function convertRequestToCurlCommand(Request $request)
      */
     public static function debugEnabled()
     {
-        return defined('DEBUG') && DEBUG;
+        return defined('ELASTICA_DEBUG') && ELASTICA_DEBUG;
```

Comment:
Hmm, haven't thought if this is BC. In that case, I would suggest to remove this `ELASTICA_DEBUG` completely, because `debug` should be an parameter of connection constructor and not some magic constant.